### ISSUE TITLE
Update Ansible to use Python 3 interpreter

### ISSUE THREAD

**Issue by universalvishwa** [dawidd6/action-ansible-playbook#9]
Hello,

This is not necessarily an issue. But, I would like to get your attention to try and use Python 3. instead of Python 2.7 as the interpreter because Python 2.7 will be deprecated very soon. It will be nice to use Python 3 as default and make it optional to use Python 2.

This will also allow to avoid those `DEPRECATION WARNINGS` when running playbooks.

Just a thought to consider. Thanks for the awesome Action you created. nice and simple.

**Comment by dawidd6** [comment]
I'm all in, but how to do that? Feel free to open a PR.

**Comment by universalvishwa** [comment]
Hi, I checked out your code. I think it might not be possible to accomplish from the way you are doing it because you use the existing Ansible installation from the OS instead of using something like the dockerized version.
 Looks like it's tied into the python 2 version during installation. This is where I found the documentation for that.
[Python 3 Support](https://docs.ansible.com/ansible/latest/reference_appendices/python_3_support.html)
